app/lib/check_elastic_search.rb

```diff
@@ -22,7 +22,6 @@ def create_elasticsearch_doc_bg(options)
     ms.attributes[:parent_id] = self.id
     ms.attributes[:created_at] = self.created_at.utc
     ms.attributes[:updated_at] = self.updated_at.utc
-    ms.attributes[:media_published_at] = self.media_published_at
```

Comment:
Question: Why you deleted this line? We're not using it anymore?

Reply:
I found this field in two places so I removed this one